dds_cli/auth.py

```diff
@@ -37,9 +37,10 @@ def __init__(
     def check(self):
         token_file = user.TokenFile(token_path=self.token_path)
         if token_file.file_exists():
-            token_file.check_token_file_permissions()
-            token = token_file.read_token()
-            token_file.token_report(token=token)
+            token = token_file.read_token(log=False)
+            token_file.token_report(
+                token=token, current_utc_time=None, expiration_time=None, log=True
+            )
```

Comment:
`current_utc_time` and `expiration_time` already have the default `None` in the `token_report` function you these are not needed here